README.md

```diff
@@ -287,11 +291,12 @@ service studentData on studentServiceListener {
         if (result is error) {
             log:printError("Error sending response", err = result);
         }
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default ootb system span.
+        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the traces. These tags are added to the default system span.